web/app/themes/mitlib-parent/inc/footer-main.php

```diff
@@ -1,23 +1,20 @@
 <?php
 /**
- * The template for the main footer, showing EDS / Barton as discovery options.
+ * The template for the main footer, showing Alma / Primo as discovery options.
  *
- * @package MITlib_Parent
- * @since 0.0.1
+ * @package MIT_Libraries_Parent
+ * @since 1.11.0
```

Comment:
I'm open to hearing that this is too fastidious, but could we revert the changes to these file tags? The values here are applicable to the legacy theme, which we're breaking from now in a way that I was hoping would be consistent across the new theme.